Initial commit

mvktsk opened the pull request.

mvktsk committed:
Initial commit

eMazeika commented on README.md:
```diff
@@ -1,2 +1,52 @@
+
+and names the folders in accordance with the rules used in the VirtoCommerce development team.
+
+Difference between the script and [Virto Commerce 2.x Module Templates](https://marketplace.visualstudio.com/items?itemName=Virto-Commerce.VirtoCommerceModuleTemplates):
```
Differences between this script

mvktsk replied:
fixed

eMazeika commented on README.md:
```diff
@@ -1,2 +1,52 @@
+## Overview
+
+This script is a lite version of [Virto Commerce 2.x Module Templates](https://marketplace.visualstudio.com/items?itemName=Virto-Commerce.VirtoCommerceModuleTemplates)
+The script creates a solution template for the VirtoCommerce platform new module. The script automatically creates all the necessary projects:
```
It generates a new Visual Studio solution for a module in Virto Commerce Platform.

mvktsk replied:
fixed

eMazeika commented on README.md:
```diff
@@ -1,2 +1,52 @@
+* Tests - xUnit Unit tests
+* Web - API Controllers
+
+and names the folders in accordance with the rules used in the VirtoCommerce development team.
```
used by the VirtoCommerce development team.

mvktsk replied:
fixed

eMazeika reviewed (changes requested):
- data project: EntityFramework package missing
- remove "Heineken" 
- Empty folders missing in Core, Data, Web projects. Can add?

mvktsk committed:
review fix

mvktsk commented:
> * data project: EntityFramework package missing
Fixed
> * remove "Heineken"
Fixed
> * Empty folders missing in Core, Data, Web projects. Can add?
Github does not save empty folders
